src/models/game.py

```diff
@@ -20,15 +45,56 @@ class Game:
     current_player_id: t.Optional[int]
     step_count: int
     winner_id: t.Optional[int]
-    field: t.Union[str, list]
+    field: t.Union[str, t.List[chr]]
     current_state: str
+    owner_mark: chr = 'X'
+    opponent_mark: chr = '0'
+
+    def _game_loop(self, field: t.List[str]):
+        for condition in WIN_CONDITIONS:
+            if all(field[cell_id] == self._current_mark() for cell_id in condition):
+                self.winner_id = self.current_player_id
+                self.current_state = State.DONE.value
+                break
+
+    def _check_win(self):
+        one_line_field = [colum for line in self.field for colum in line]
+        if one_line_field.count('') == 0:
```

Comment:
If i understand aright, here you checks if there is no empty cells - it mean that nobody wins, but you allready check if there is step_count >= fiel_size that do the same thing, but without loop, that more optimized